src/lib/storage/proposal/partition_creator.rb

```diff
@@ -230,6 +230,8 @@ def create_partition(vol, partition_id, free_slot)
             partition
           # FIXME: rescue ::Storage::Exception when SWIG bindings are fixed
           rescue RuntimeError => ex
+            # Don't hide our own exceptions
+            raise if ex.is_a?(Yast::Storage::Proposal::Error)
```

Comment:
maybe less tricky to let ruby exception handler to recognize it, so something like
```
# Don't hide our own exceptions
rescue Yast::Storage::Proposal::Error
  raise
rescue RuntimeError => ex
  ...
```